### PR TITLE
Login & Signup fix 

### LINKED ISSUE #67: Commune Page
Page which presents commune data

### PR DESCRIPTION
**Pull request by RonTuretzky**
Fix #67

### CODE REVIEW

File: commune_app/views.py [hunk 1]
```diff
@@ -31,12 +38,13 @@ def user_signup(request):
     else:
         return render(request, 'commune_app/signup.html')
 
+def user_login(request):
```
Comment (kobihk) [inline commune_app/views.py:41]: you have this line twice - here and in line # 43
please delete this line 
```suggestion
```

**Review by kobihk — CHANGES_REQUESTED** [review]

**Review by EliasAndry — APPROVED** [review]

**Review by NicolePauzner — APPROVED** [review]

**Review by FaezZangariya — DISMISSED** [review]

**Review by ofirtzabari — DISMISSED** [review]
amazing

File: commune_app/templates/base.html [hunk 2]
```diff
@@ -24,10 +24,28 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (kobihk) [inline commune_app/templates/base.html:48]: Please remove all the empty lines
```suggestion
```

File: commune_app/templates/commune_app/login.html [hunk 3]
```diff
@@ -42,4 +42,4 @@
 <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.3.1/dist/js/bootstrap.min.js" integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM" crossorigin="anonymous"></script>
 </body>
 </html>
-{% endblock %}
+{% endblock %}
```
Comment (kobihk) [inline commune_app/templates/commune_app/login.html:45]: please add a new line

File: commune_app/templates/commune_app/logout.html [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+    <meta charset="UTF-8">
+    <title>Title</title>
+      Hi {{ user.username }}!
+  <p><a href="{% url 'logout' %}">Log Out</a></p>
+
+</head>
+<body>
+
+</body>
+</html>
```
Comment (kobihk) [inline commune_app/templates/commune_app/logout.html:13]: please add a new line

File: commune_app/templates/commune_app/logout.html [hunk 5]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```
Comment (kobihk) [inline commune_app/templates/commune_app/logout.html:1]: I think the logout should return to the main page 
IMHO we don't need this page

**Review by kobihk — CHANGES_REQUESTED** [review]

File: commune_app/templates/base.html [hunk 6]
```diff
@@ -24,10 +24,28 @@
...
-            <li class="nav-item"><a href="{% url 'Main_Page' %}" class="nav-link active text-white">Home</a></li>
+            <li class="nav-item"><a href="{% url 'main_page' %}" class="nav-link active text-white">Home</a></li>
             <li class="nav-item"><a href="#story" class="nav-link active text-white">About Us</a></li>
 			<li class="nav-item"><a href="{% url 'signup' %}" class="nav-link active text-white">Join Us</a></li>
-            <li class="nav-item"><a href="{% url 'commune' %}" class="nav-link active text-white">Communes</a></li></ul>
+            <li class="nav-item"><a href="{% url 'commune' %}" class="nav-link active text-white">Communes</a></li>
+            {% if request.user.is_authenticated %}
+                <li class="nav-item"><a href="{% url 'logout' %}" class="nav-link active text-white">Logout {{ request.user.username }}</a></li>
+            {% else %}
+                    <li class="nav-item"><a href="{% url 'logout' %}" class="nav-link active text-white">Logout</a></li>
```
Comment (kobihk) [inline commune_app/templates/base.html:34]: if the user is NOT authenticated 
I think you can remove this line - you don't need logout when the user is not logged in
```suggestion
```

**Review by kobihk — CHANGES_REQUESTED** [review]

**Review by EliasAndry — APPROVED** [review]
perfectooo

**Review by FaezZangariya — APPROVED** [review]

**Review by kobihk — APPROVED** [review]
Please open an issue to add tests for the pages in different PR

**Pull request merged**